wiki/de/de-Setup.md

```diff
@@ -11,7 +11,7 @@ Um das Beste von Jamulus genießen zu können, brauchst du mindestens:
 
 1. **Eine kabelgebundene Internetverbindung** (kein WLAN - siehe [FAQ](/wiki/FAQ))
 1. **Kabelgebundene Kopfhörer (weder Bluetooth noch Lautsprecher** - siehe [FAQ](/wiki/FAQ))
-1. **Ein vernünftiges Audiointerface, Soundkarte und Mikrofon** ([siehe diese Auflistung](/kb/2021/01/05/Jamulus-Sound-Devices.html) für Beispiele)
+1. **Ein passendes Audiointerface oder Soundkarte und Mikrofon** ([siehe diese Auflistung](/kb/2021/01/05/Jamulus-Sound-Devices.html) für Beispiele)
```

Comment:
I think this goes with style and tone: See https://github.com/jamulussoftware/jamuluswebsite/issues/339 which is for english. But I think we can leave it like that for the moment.